sympy/stats/rv_interface.py

```diff
@@ -202,8 +203,9 @@ def cmoment(X, n, condition=None, **kwargs):
     >>> cmoment(X, 2) == variance(X)
     True
     """
-    mu = expectation(X, condition, **kwargs)
-    return moment(X, n, mu, condition, **kwargs)
+    evaluate = kwargs.pop('evaluate', True)
+    mu = expectation(X, condition, evaluate=evaluate, **kwargs)
+    return moment(X, n, mu, condition, evaluate=evaluate, **kwargs)
```

Comment:
Why pop evaluate from the kwargs if it is only going to be passed along with the kwargs?